test.xml

```diff
@@ -0,0 +1,4560 @@
+<dependencies>
```

Comment:
I think we can delete this file...looks like it was something that I accidentally uploaded when I committed some changes.